config/core_pools.json

```diff
@@ -26,11 +26,11 @@
       "0x05ff47afada98a98982113758878f9a8b9fdda0a000000000000000000000645": "weETH/rETH"
     },
     "polygon": {
-      "0xf0ad209e2e969eaaa8c882aac71f02d8a047d5c2000200000000000000000b49": "ECLP-WMATIC-stMATIC",
+      "0x8159462d255c1d24915cb51ec361f700174cd99400000000000000000000075d": "ECLP-WMATIC-stMATIC",
```

Comment:
the old one has $5.7m tvl, the new one only $1.1m?

Reply:
The new one is not gyro pool as well?

Reply:
aaah it should be replacing the non gyro on L31